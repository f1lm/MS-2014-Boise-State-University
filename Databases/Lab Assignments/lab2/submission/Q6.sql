select DNAME from (
select DName, count(distinct C.CourseNo) as Course from dept D
inner join course C on C.DeptCode = D.DeptCode
inner join section Sec on Sec.CourseNo = C.CourseNo 
where Sec.Year='2001' and Sec.Semester='S' group by D.DName) as temp
where Course >=2;