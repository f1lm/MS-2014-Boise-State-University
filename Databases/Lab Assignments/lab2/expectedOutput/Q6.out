mysql> source Q6.sql;
+------------------+
| DNAME            |
+------------------+
| Computer Science |
+------------------+
1 row in set (0.00 sec)

mysql> notee;
